lib/file.js

```diff
@@ -78,7 +78,7 @@ fuse(File, require('eventemitter3'));
 File.readable('location', {
   enumerable: false,
   get: function get() {
-    if (this.external) return this.filepath;
+    if (this.external) return this.filepath + this.extname;
     return '/' + this.hash + this.extname;
```

Comment:
ext name gets cut off of filepath, so full http://my.js, is just http://my when filepath is returned